stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -31,3 +32,28 @@ logs = (start, stop=now(), fn) =>
 deadman = (t, tables=<-) => tables
     |> max(column: "_time")
     |> map(fn: (r) => ( {r with dead: r._time < t} ))
+
+levelOK = 0
+levelInfo = 1
+levelWarn = 2
+levelCrit = 3
+levelUnknown = -1
+
+// Check performs a check against its input using the given ok, info, warn and crit functions
+// and writes the result to a system bucket.
+check = (tables=<-, check, messageFn, crit, warn, info, ok) =>
+    tables
+        |> experimental.set(o: check.tags)
```

Comment:
Is there a reason we don't just add an extra option to `set` so that it sets them as tags? Either of these two:

```
// an optional parameter to say that the data is going to be added to the group key
experimental.set(data: check.tags, key: true)

// use data to set normal data and tags to set tags
experimental.set(tags: check.tags)
```

Reply:
Personally I like having smaller composable operations, but since adding such an option would never make it so that we need to regroup, I'd be okay with it.

Reply:
That's mostly my perspective on it. The primary reason is performance, but also just that it requires less code to get the desired effect.